sw/host/opentitantool/h1dx_devboard.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
All of these json files should go into a subdirectory with a name like `config` or `resources`.